src/Raven.Server/ServerWide/ServerVersion.cs

```diff
@@ -18,6 +20,9 @@ public class ServerVersion
             _commitHash ?? (_commitHash = RavenVersionAttribute.Instance.CommitHash);
         public static string FullVersion => 
             _fullVersion ?? (_fullVersion = RavenVersionAttribute.Instance.FullVersion);
+        public static string SemVersion =>
+            _semVersion ?? (_semVersion = Version + ".0." + Build);
```

Comment:
this is wrong, after releasing 4.0.0 stable, next stable will be 4.0.1, but here you have hardcoded .0 so it will be still 4.0.0.

The proper string is in 'FullVersion' property, check here: http://4.live-test.ravendb.net/build/version
"FullVersion": "4.0.0-alpha-40008"
Just use that in WelcomeMessage